src/components/Admin/SubscriptionDetails.jsx

```diff
@@ -37,7 +37,7 @@ const SubscriptionDetails = ({ enterpriseSlug }) => {
           <div className="mt-3 d-flex align-items-center">
             <div className="mr-5">
               <span>
-                {moment(subscription.startDate).format('MMMM D, YYYY')} - {moment(subscription.expirationDate).format('MMMM D, YYYY')}
+                {dayjs(subscription.startDate).format('MMMM D, YYYY')} - {dayjs(subscription.expirationDate).format('MMMM D, YYYY')}
```

Comment:
I'm curious why you're using a date library for this at all, instead of `react-intl`'s [`<FormattedDate>`](FormattedDate) component, which I believe is already included as a paragon peer dependency? (And uses the native `Intl` API behind the scenes, so has very little code size overhead).